Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ruby:3.2.1
+
+RUN apt update -qq && \
+    apt install -y build-essential curl git wget
+
+RUN wget -O duckdb.zip "https://github.com/duckdb/duckdb/releases/download/v0.6.1/libduckdb-linux-amd64.zip"
```

Comment:
It is ok that default Ruby version is 3.1 and duckdb version 0.6.1.
But is there any way to specify those versions from command line option?

Reply:
OK I will try using --build-arg option